### ISSUE TITLE
Bug in Org-babel  support in emacs

### ISSUE THREAD

**Issue by OmarAshkar** [donkirkby/live-py-plugin#299]
Hello,

This is related to #259.

I installed live-py-mode from melpa and I cannot work in org-babel 
Step to reproduce: 

1. Open org file 
2. Start a python src

```
#+begin_src python
#+end_src
```
3. inside the src execute edit-src-special C-c C-'
3. activate py-live-mode
The mode will not be activated resulting in a message 

```
Current buffer has no associated file
```

It is worth noting I am using doom emacs. I think the answer is to determine when the package should be sequentially to get this value.

# My environment
Describe the versions of everything you were using:
* EMACS 
* Operating system Ubuntu 20.04
* Python 3.8.5

Thank You

**Comment by donkirkby** [comment]
Yes, I can reproduce the problem with Emacs 27.2 on Ubuntu 20.04. Is that the same version that you're running?
Thanks for the bug report.

**Comment by OmarAshkar** [comment]
@donkirkby  I am using 27.1, so shouldn't  be drastic
